File: python/7 kyu/28.py
# https://www.codewars.com/kata/5785cd91a1b8d5c06e000007/train/python

# Passed

def re_ordering(text):
    words = text.split()
    capital_word = [w for w in words if not w.islower()][0]
    capital_word_index = words.index(capital_word)

    words_reordered = [capital_word] + words[:capital_word_index] + words[capital_word_index + 1:]
    return ' '.join(words_reordered)

output = re_ordering('pl mx Lyvyclg lirofr lpzqb hmjpc')
print(output)

File: nodejs/7 kyu/28.js
// https://www.codewars.com/kata/52b5247074ea613a09000164/train/javascript

// Passed

function cookingTime(eggs) {
    let eggBatches = eggs % 8 ? Math.floor(eggs/8) + 1 : eggs / 8;
    return 5  * eggBatches;
}

const output = cookingTime(17);
console.log(output);